### ISSUE TITLE
Review/edit Dev Guide > Python style guide

### ISSUE THREAD

**Issue by sarina** [openedx/docs.openedx.org#413]
Review https://docs.openedx.org/en/latest/developers/references/legacy_guide/style_guides/python-guidelines.html for organization and clarity

**Comment by MaferMazu** [comment]
Hello @sarina, the link to the legacy doc is broken. Do you have another, or is it lost?

**Comment by sarina** [comment]
Yes, that is it. Hmm... I must have renamed the "legacy guide" without a redirect
